src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -237,83 +316,110 @@ const CurationPage = (props: ICurationPageProps) => {
+                                          mutationIndex,
+                                          tumorIndex,
+                                          tiIndex,
+                                          treatmentIndex
+                                        )}
+                                      >
+                                        <RealtimeDropdownInput
+                                          fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/level`}
+                                          label="Highest level of evidence"
+                                          name="level"
+                                          options={[TX_LEVELS.LEVEL_NO, TX_LEVELS.LEVEL_1, TX_LEVELS.LEVEL_2]}
+                                        />
+                                        <RealtimeDropdownInput
+                                          fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/propagation`}
+                                          label="Level of Evidence in other solid tumor types"
+                                          name="propagationLevel"
+                                          options={[]} // Todo
+                                        />
+                                        <RealtimeDropdownInput
+                                          fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/propagationLiquid`}
+                                          label="Level of Evidence in other liquid tumor types"
+                                          name="propagationLiquidLevel"
+                                          options={[]}
+                                        />
+                                        <RealtimeDropdownInput
+                                          fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/fdaLevel`}
+                                          label="FDA Level of Evidence"
+                                          name="propagationLiquidLevel"
+                                          options={[]}
+                                        />
+                                        <RealtimeTextAreaInput
+                                          fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/description`}
+                                          inputClass={styles.textarea}
+                                          label="Description of Evidence"
+                                          labelIcon={
+                                            <GeneHistoryTooltip
+                                              historyData={parsedHistoryList}
+                                              location={`${CANCER_TYPE_THERAPY_INDENTIFIER}${getMutationName(
```

Comment:
The tooltip looks for that identifier since the location for cancer type has so many variations. When the tooltip sees this identifier it knows to do some special parsing for the location. I could have just checked explicitly for the character represented by that constant (&), but if the identifier passed into the tooltip were changed, the logic for parsing it would need to reflect that change in the tooltip. By making the identifier a constant, any changes to the identifier will not require changes to the parsing logic.